CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endif()
 
 option(LIBEVSE_SECURITY_USE_BOOST_FILESYSTEM "Usage of boost/filesystem.hpp instead of std::filesystem" OFF)
 
+option(LIBEVSE_CRYPTO_SUPPLIER_OPENSSL "Default OpenSSL cryptography supplier" ON)
```

Comment:
Note: This section of code will need to be revised when multiple suppliers are added (OpenSSL should be the default, but the user shouldn't need to explicitly disable it if another supplier is added)